portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,33 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
```

Comment:
We generally prefer to keep styles separate from the html, so best to move this to the css file. See the style guide for some more info here, https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns.